webrtc.html

```diff
@@ -1908,8 +1908,25 @@ <h4>Set the RTCSessionDescription</h4>
                   </li>
                   <li data-tests="RTCPeerConnection-onnegotiationneeded.html">
                     <p>If <var>connection</var>'s [= signaling state =] is now
-                    {{RTCSignalingState/"stable"}}, [= clear the negotiation-needed
-                    flag =] and [= update the negotiation-needed flag =].
+                    {{RTCSignalingState/"stable"}}, run the following steps:</p>
+                    <ol>
+                      <li>
+                        <p>For any <var>transceiver</var> that was removed from
+                        the [= set of transceivers =] in a previous step, if any
+                        of its transports
+                        (<var>transceiver</var>.<a>[[\Sender]]</a>.<a>[[\SenderTransport]]</a>
+                        or <var>transceiver</var>.<a>[[\Receiver]]</a>.<a>[[\ReceiverTransport]]</a>)
```

Comment:
Unsure about "for any". I worry it's not clear how many times we close the same transport, or what order events fire. Maybe I'm being pedantic but how about:

- Let *transports* be an empty set.
- For each _transceiver_ that was removed from the _set of transceivers_ in a previous step, add  _transceiver_.[[Sender]].[[SenderTransport]] and _transceiver_.[[Receiver]].[[ReceiverTransport]] to _transports_.
- For each _transport_ in _transports_, **close the RTCDtlsTransport** _transport_.

Then add a **close the RTCDtlsTransport** algorithm somewhere near [here](https://w3c.github.io/webrtc-pc/#dfn-dtlstransportstate) that takes care of firing `statechange` and then queues a task to close its `transport.iceTransport` (or maybe in the same task like https://github.com/w3c/webrtc-pc/pull/2444?) 

Then to avoid confusion, amend this text [near here](https://w3c.github.io/webrtc-pc/#dfn-dtlstransportstate) (addition in **bold**):

"When the underlying DTLS transport needs to update the state of the corresponding RTCDtlsTransport object for any other reason, **and the new state is not `"closed"`,**"

Reply:
Due to a lack of time I did a lazy version of making sure its not closed twice: I added the caveat "are still not closed". But I can do the pedantic version of this later if you think it is needed.